test/IRGen/constant_struct_with_padding.sil

```diff
@@ -0,0 +1,26 @@
+// RUN: %swift -module-name main %s -emit-ir -o - | %FileCheck %s
```

Comment:
Ah, this RUN line turns out to cause problems when cross-compiling, where there's no host stdlib. Can you make the test use `%target-swift-frontend` instead, and generalize it enough so that it passes on all the interesting platforms?

(There's a